addon/components/offering-form.js

```diff
@@ -439,28 +446,40 @@ export default Component.extend(ValidationErrorDisplay, Validations, {
 
     yield this.get('saveOffering').perform();
   }),
-  updateDurationMinutes: task(function * (minutes) {
-    let {validations} = yield this.validate();
-    this.send('addErrorDisplayFor', 'durationMinutes');
 
-    if (validations.get('durationMinutes.isInvalid')) {
+  updateDurationHours: task(function * (hours) {
+    yield timeout(DEBOUNCE_DELAY);
+    this.set('hourBuffer', hours);
```

Comment:
I think what you're trying to do here is to validate the data without changing the value, the problem is that this invalidates the work done on save. This isn't a good solution if it doesn't allow validations to work the same in all tasks.